CMakeLists.txt

```diff
@@ -149,33 +149,39 @@ if (SCN_INSTALL)
 
     install(DIRECTORY
             "${CMAKE_CURRENT_SOURCE_DIR}/include/"
-            DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+            DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+            COMPONENT Development)
```

Comment:
Please prefix these with a project unique string as well, so the component names work well in vendoring scenarios. I would recommend something like `scnlib_Development`.

Reply:
Done